README.md

```
@@ -0,0 +1,117 @@
+It is built on top of PyTorch and provides a set of tools for training and evaluating deep learning models using the factory pattern[^1]. 
+So you can easily build your own model and train it with your own dataset by adding a new class to the modules.
+
+HePyporch is currently focusing on the following tasks:
+- Estimation of the top quark mass based from the kinematic properties of the top quark decay products based on the Monte Carlo simulation
+- B-jet matching between the reconstructed jets and the leptons from the top quark decay
+
+## Installation
+### Using Pip
+With the option -e is used to install the package in editable mode, so you can modify the code and see the changes immediately without reinstalling the package. 
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)